### PR TITLE
Add TeamPaper Latest

### PR DESCRIPTION
**Pull request by cema-sp**
Add TeamPaper Snap screen capturing tool (https://teampaper.me)

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/teampaper.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'teampaper' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://teampaper.me/snap/download/teampapersnap.dmg'
```
Comment (miccal) [inline Casks/teampaper.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).You will also need to update the `version` and `sha256` accordingly.

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/teampaper.rb (hunk 1 above)
Comment (cema-sp) [inline Casks/teampaper.rb]: I could add `version` and `sha256`, but the project doesn't have any appcast 😞

Comment (miccal) [inline Casks/teampaper.rb]: Please run the command suggested in the [`appcast` documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) - you will find this app _does_ have an `appcast`.

Comment (cema-sp) [inline Casks/teampaper.rb]: @miccal thank you! It is the first time I heart of Sparkle

Comment (vitorgalvao) [inline Casks/teampaper.rb]: > @miccal thank you! It is the first time I heart of Sparkle

You know when you’re using a native macOS app and it prompts you for an update? Notice how most of the time the update process looks exactly the same for different apps? That’s [sparkle](https://sparkle-project.org/).

**Review by vitorgalvao — APPROVED** [review]

File: Casks/teampaper.rb (hunk 1 above)
Comment (cema-sp) [inline Casks/teampaper.rb]: @vitorgalvao Yes, I've already seen that update window in various apps

**Pull request merged**